From e58fe80bcb86711ecfb9e3083b3876e5db320a05 Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexdeucher@gmail.com>
Date: Tue, 27 Jul 2010 11:11:19 -0400
Patch-mainline: 2.6.36
References: fate#310916
Git-commit: 7fc8878c3398a71c39e23ae1d9d56ba1f9e8c97d
Subject: [PATCH 0977/2588] drm/radeon: remove duplicate state emit in
 r6xx/r7xx blit

Signed-off-by: Alex Deucher <alexdeucher@gmail.com>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit 7fc8878c3398a71c39e23ae1d9d56ba1f9e8c97d)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/radeon/r600_blit_shaders.c |   16 ----------------
 1 files changed, 0 insertions(+), 16 deletions(-)

diff --git a/drivers/gpu/drm/radeon/r600_blit_shaders.c b/drivers/gpu/drm/radeon/r600_blit_shaders.c
index a8f8bbc..3bcbb93 100644
--- a/drivers/gpu/drm/radeon/r600_blit_shaders.c
+++ b/drivers/gpu/drm/radeon/r600_blit_shaders.c
@@ -459,10 +459,6 @@ const u32 r6xx_default_state[] =
 	0x00000000,
 
 	0xc0016900,
-	0x00000206, /* PA_CL_VTE_CNTL */
-	0x00000000,
-
-	0xc0016900,
 	0x00000207,
 	0x00000000, /* PA_CL_VS_OUT_CNTL */
 
@@ -478,10 +474,6 @@ const u32 r6xx_default_state[] =
 	0x3f800000,
 
 	0xc0016900,
-	0x00000205,
-	0x00000004, /* PA_SU_SC_MODE_CNTL */
-
-	0xc0016900,
 	0x00000280,
 	0x00000000, /* PA_SU_POINT_SIZE */
 
@@ -1118,10 +1110,6 @@ const u32 r7xx_default_state[] =
 	0x00000000,
 
 	0xc0016900,
-	0x00000206,
-	0x00000000, /* PA_CL_VTE_CNTL */
-
-	0xc0016900,
 	0x00000207,
 	0x00000000, /* PA_CL_VS_OUT_CNTL */
 
@@ -1137,10 +1125,6 @@ const u32 r7xx_default_state[] =
 	0x3f800000,
 
 	0xc0016900,
-	0x00000205,
-	0x00000004, /* PA_SU_SC_MODE_CNTL */
-
-	0xc0016900,
 	0x00000280,
 	0x00000000, /* PA_SU_POINT_SIZE */
 
-- 
1.7.6

